atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -32,17 +29,13 @@ class SendMailPlugin(ExitPlugin):
                 "args": {
                     "send_on": ["auto_canceled", "auto_fail"],
                     "url": "https://openshift-instance.com",
-                    "pdc_url": "https://pdc-instance.com",
-                    # pdc_secret_path is filled in automatically by osbs-client
-                    "pdc_secret_path": "/path/to/file/with/pdc/token",
                     "smtp_uri": "smtp-server.com",
                     "from_address": "osbs@mycompany.com",
                     "error_addresses": ["admin@mycompany.com"],
                     # optional arguments follow
```

Comment:
Missing `email_domain`.
Maybe we should remove this "optional arguments follow" comment, since all params in this plugin are optional.
Or, make the ones above this line, not optional.